packages/simcore-sdk/src/simcore_sdk/node_ports_common/_filemanager.py

```diff
@@ -66,10 +73,11 @@ async def _complete_upload(
             Envelope[FileUploadCompleteResponse], await resp.json()
         )
         assert file_upload_complete_response.data  # nosec
-    state_url = file_upload_complete_response.data.links.state
+    state_url = _get_secure_link(file_upload_complete_response.data.links.state)
     _logger.info(
-        "completed upload of %s",
+        "completed upload of %s\n using %s",
```

Comment:
Q: you want to drop a large json in an INFO. Should it bea debug instead?

Reply:
I will leave it as is in info, but I will print the correct url instead and make it not complicated.
It was always 1 link